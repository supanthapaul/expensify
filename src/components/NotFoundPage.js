import React from 'react'
import { Link } from 'react-router-dom'

const NotFoundPage = () => (
    <div>
        <p>404! Page not found - <Link to="/">Go Home</Link> </p>
    </div>
);

export default NotFoundPage;