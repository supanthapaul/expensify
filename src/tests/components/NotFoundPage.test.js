import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../components/NotFoundPage';

test('Should render NotFoundPage correctly', () => {
   // Enzyme docs - http://airbnb.io/enzyme/docs/api/shallow.html
   const wrapper = shallow(<NotFoundPage />);
   expect(wrapper).toMatchSnapshot();
});
