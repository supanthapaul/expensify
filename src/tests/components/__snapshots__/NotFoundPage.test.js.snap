// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render NotFoundPage correctly 1`] = `
<div>
  <p>
    404! Page not found - 
    <Link
      replace={false}
      to="/"
    >
      Go Home
    </Link>
     
  </p>
</div>
`;
